src/crimson/os/seastore/transaction.h

```diff
@@ -72,6 +73,7 @@ struct version_stat_t {
  * - seastore_cache logs
  */
 class Transaction {
+  static transaction_id_t next_id;
```

Comment:
This can't be a static variable.  We'll shortly be trying to run multiple seastore instances within a single process and after that we'll need to generalize sesatore's structures to partition them across cores.  This should probably be owned by whatever is currently constructing Transactions and passed into the Transaction constructor.

Reply:
yeah, will change this part